Source/BugsnagKeys.h

```diff
@@ -43,6 +43,7 @@ static NSString *const BSGKeyLabel = @"label";
 static NSString *const BSGKeySeverityReason = @"severityReason";
 static NSString *const BSGKeyLogLevel = @"logLevel";
 static NSString *const BSGKeyOrientation = @"orientation";
+static NSString *const BSGKeyOrientationChange = @"Orientation change";
```

Comment:
I'm not sure this is really a key - I think it should use the notification name.

Reply:
It's not a key per-se, but it is a constant string.  `keys.h` was an unobtrusive place to put it.  As per other constant strings scattered throughout the library I'll put it in `client.m`.

Reply:
Resolved after discussion.  Removed, replaced by a key in `notificationNameMap`.